backend/src/zac/core/rules.py

```diff
@@ -189,6 +189,19 @@ def is_zaak_behandelaar(user: User, zaak: Optional[Zaak]):
     return bool(user_rollen)
 
 
+@rules.predicate
+def is_zaak_adviser(user: User, zaak: Optional[Zaak]):
```

Comment:
```suggestion
def is_zaak_reviewer(user: User, zaak: Optional[Zaak]):
```
reviewer is the more generic form of advisor/approver